src/internal/orders/order-app.ts

```diff
@@ -61,7 +61,7 @@ export class OrderApp extends ConnectionApp {
   }
 
   public async getSalesOrdersByDate?(
-    transaction: TransactionPOJO, range: SalesOrderTimeRangePOJO): Promise<SalesOrderArray[]> {
+    transaction: TransactionPOJO, range: SalesOrderTimeRangePOJO): Promise<SalesOrderArray> {
```

Comment:
Instead of SalesOrderArray why not just name it SalesOrders. The Array suffix just makes me think that it is an array that can be iterated on. Unless the plan is to provide an iterator for this type.

Reply:
It _is_ an array.

Reply:
Yea, you can iterate it and `.map` it and stuff.

Reply:
@justin-shipstation, yeah, I wrote it to where it extends the base `Array` type and adds the `paging` property to it.